src/pages/my_page/user_profile/UserRoleAndHelp.tsx

```diff
@@ -0,0 +1,118 @@
+import LaunchIcon from '@mui/icons-material/Launch';
+import { Box, Divider, Link as MuiLink, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { fetchUser } from '../../../api/roleApi';
+import { BackgroundDiv } from '../../../components/styled/Wrappers';
+import { RootState } from '../../../redux/store';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { ViewingScopeChip } from '../../basic_data/institution_admin/edit_user/ViewingScopeChip';
+import { UserRoles } from './UserRoles';
+
+export const UserRoleAndHelp = () => {
+  const { t } = useTranslation();
+  const user = useSelector((store: RootState) => store.user);
+  const customer = useSelector((store: RootState) => store.customer);
+  const customerServiceCenterUri = customer?.serviceCenterUri;
+
+  const username = user?.nvaUsername ?? '';
+
+  const nvaUserQuery = useQuery({
+    enabled: !!username,
+    queryKey: [username],
+    queryFn: () => fetchUser(username),
+    meta: { errorMessage: t('feedback.error.get_person') },
+  });
+
+  const nvaUser = nvaUserQuery.data;
+
+  return (
+    <BackgroundDiv sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+      {nvaUser?.viewingScope && (
+        <Box sx={{ gap: '0.75rem' }}>
```

Comment:
her settes gap uten at det er noen flex eller grid, da har den ingen effekt (?)
```suggestion
        <div>
```